docs/topics/logging.txt

```diff
@@ -398,12 +408,43 @@ configure logging using your own approach), you can set
 :setting:`LOGGING_CONFIG` to ``None``. This will disable the
 configuration process.
 
-.. note::
-    Setting :setting:`LOGGING_CONFIG` to ``None`` only means that the
-    configuration process is disabled, not logging itself. If you
-    disable the configuration process, Django will still make logging
-    calls, falling back to whatever default logging behavior is
-    defined.
+Setting :setting:`LOGGING_CONFIG` to ``None`` only means that the configuration
+process is disabled, not logging itself. If you disable the configuration
+process, Django will still make logging calls, falling back to whatever default
+logging behavior is defined.
+
+Here's an example of how to make the logging system print everything to the
+console without having Django's disabled default loggers swallow entries. It
+may be useful during local development. Set the environment variable
+``DJANGO_LOG_LEVEL=DEBUG`` to see all of Django's logging calls in the console.
+
+.. snippet::
```

Comment:
This example may be useful to include, but it does not belong in this section. You can use this config to get useful console logging while leaving Django's default logging config also in place, so use of this configuration is orthogonal to the question of setting `LOGGING_CONFIG` to `None` to disable Django's default logging config.